src/utils/auth.js

```diff
@@ -129,7 +125,10 @@ class Auth {
 
   renewSession() {
     return new Promise(async (resolve, reject) => {
-      if (window.location.hash) {
+      if (typeof window === 'undefined') {
```

Comment:
I fill that we should have a module to make this repeated piece more readable (`isSsr`?).

Reply:
#884

Reply:
That sounds good to me.  There's lots of quality-of-life stuff to be done for sure.  Good idea to start now.